models/marketing/email_events/email_events.yml

```diff
@@ -20,8 +20,6 @@ models:
       - name: event_id
         description: The ID of the event.
         tests:
-          - dbt_expectations.expect_column_values_to_be_unique:
```

Comment:
Wondering if we replace the dbt_expectations macro with a unique test

Reply:
update -- this is more complex than what a unique test can offer so will open up a FR to create a macro for this.